entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EuropeanaEntityLd.java

```diff
@@ -0,0 +1,345 @@
+package eu.europeana.entitymanagement.utils.jsonld;
+
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.stanbol.commons.jsonld.JsonLd;
+import org.apache.stanbol.commons.jsonld.JsonLdProperty;
+import org.apache.stanbol.commons.jsonld.JsonLdPropertyValue;
+import org.apache.stanbol.commons.jsonld.JsonLdResource;
+
+import eu.europeana.api.commons.definitions.utils.DateUtils;
+import eu.europeana.entitymanagement.definitions.exceptions.UnsupportedEntityTypeException;
+import eu.europeana.entitymanagement.definitions.model.Agent;
+import eu.europeana.entitymanagement.definitions.model.Concept;
+import eu.europeana.entitymanagement.definitions.model.ConceptScheme;
+import eu.europeana.entitymanagement.definitions.model.Entity;
+import eu.europeana.entitymanagement.definitions.model.Organization;
+import eu.europeana.entitymanagement.definitions.model.Place;
+import eu.europeana.entitymanagement.definitions.model.Timespan;
+import eu.europeana.entitymanagement.definitions.model.impl.BaseEntity;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.EntityTypes;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.WebEntityConstants;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.WebEntityFields;
+import eu.europeana.entitymanagement.utils.EntityUtils;
+
+
+public class EuropeanaEntityLd extends JsonLd {

```

Comment:
we will not use Annotation-ld library for serilization but Jackson, please remove this class. See also the JsonLdSerializer class in branch 2326

Reply:
Removed.